breadcord/module.py

```diff
@@ -69,17 +69,18 @@ def load_settings_schema(self) -> None:
         settings.in_schema = True
 
     def install_requirements(self) -> None:
-        distributions = list(importlib.metadata.distributions())
-        if missing_requirements := [
-            str(requirement)
-            for requirement in self.manifest.requirements
-            if not any(
-                requirement.name == distribution.name
-                and distribution.version in requirement.specifier
-                for distribution in distributions
-            )
-        ]:
-            self.logger.info(f'Installing missing requirements: {", ".join(missing_requirements)}')
+        installed_distributions = tuple(importlib.metadata.distributions())
+
+        def is_installed(requirement: Requirement) -> bool:
+            for distribution in installed_distributions:
+                if requirement.name == distribution.name and distribution.version in requirement.specifier:
+                    return True
+            return False
+
+        if missing_requirements := filter(
+            lambda requirement: not is_installed(requirement),
+            self.manifest.requirements,
+        ):
```

Comment:
Why `is_installed` instead of `is_missing`? The latter would be neater and remove the need for a lambda altogether